### PR TITLE
Added build script to auto increment build number

### PR DESCRIPTION
**Pull request by petermisik**

### CODE REVIEW

File: Logging/Controllers/View Controllers/Form 781/FlightListViewController.swift [hunk 1]
```diff
@@ -316,6 +317,43 @@ class FlightListViewController: UIViewController {
...
+            NSLog("Time is valid")
+        } catch Form781Error.InvalidHours {
+            Alerts.showHoursError(on: self)
+            Helper.highlightRed(textField: takeOffTime)
+        } catch Form781Error.InvalidMins {
+            Alerts.showMinError(on: self)
+            Helper.highlightRed(textField: takeOffTime)
+        } catch Form781Error.NoTimeFound {
+            Helper.highlightRed(textField: takeOffTime)
+            NSLog("No time found")
```
Comment (Pete-Hoch) [inline Logging/Controllers/View Controllers/Form 781/FlightListViewController.swift:350]: landTime?

Or better yet, can this be refactored into a single function that takes the textField in as a parameter?

File: Logging/HelperFiles/HelperFunctions.swift [hunk 2]
```diff
@@ -75,6 +74,42 @@ class Helper {
...
+
+         - Parameter timeString: String - Represents the time to test
+
+         - Throws: Form781Error.InvalidTime
+
+         - Returns: Bool Valid time
+         
+            Just a simple function to validate the hours and minutes
+         */
+
```
Comment (Pete-Hoch) [inline Logging/HelperFiles/HelperFunctions.swift:90]: Can the documentation go just on the outside of the generateImage function?

File: Logging/HelperFiles/HelperFunctions.swift [hunk 3]
```diff
@@ -259,6 +294,18 @@ class Helper {
...
+             - Parameter none: All data is being retrieved from the stored JSON.
+
+             - Throws: none
+
+             - Returns: An image that can be rendered with the printController.
+             
+                throughout the function, any hard coded numbers represent pixels on the underlay image.  We use an NSAttributedString which gives the ability to control the font size.  Then we use the draw function to position it on the page.
+             */
+
```
Comment (Pete-Hoch) [inline Logging/HelperFiles/HelperFunctions.swift:308]: This looks out of place too.

**Review by Pete-Hoch — CHANGES_REQUESTED** [review]
There seems to be a log of thrash in the project file that I do not understand.

Just a few comments on the source code.

File: Logging.xcodeproj/project.pbxproj [hunk 4]
```diff
@@ -31,7 +31,9 @@
 		130D721D255AD7DB0025C589 /* afto781.pdf in Resources */ = {isa = PBXBuildFile; fileRef = 131239A22518EFCF00F3B636 /* afto781.pdf */; };
 		131239A32518EFCF00F3B636 /* afto781.pdf in Resources */ = {isa = PBXBuildFile; fileRef = 131239A22518EFCF00F3B636 /* afto781.pdf */; };
 		131239A82518F98100F3B636 /* afto781.jpg in Resources */ = {isa = PBXBuildFile; fileRef = 131239A72518F98000F3B636 /* afto781.jpg */; };
-		13941316254B4C4000B673FE /* Form781Error.swift in Sources */ = {isa = PBXBuildFile; fileRef = 13941315254B4C4000B673FE /* Form781Error.swift */; };
+		13673C902566D50B00733936 /* Form781Error.swift in Sources */ = {isa = PBXBuildFile; fileRef = 13673C8F2566D50B00733936 /* Form781Error.swift */; };
+		13673C912566D50B00733936 /* Form781Error.swift in Sources */ = {isa = PBXBuildFile; fileRef = 13673C8F2566D50B00733936 /* Form781Error.swift */; };
+		13673C922566D50B00733936 /* Form781Error.swift in Sources */ = {isa = PBXBuildFile; fileRef = 13673C8F2566D50B00733936 /* Form781Error.swift */; };
```
[the remark this answers is not in the document]
Comment (petermisik) [reply Logging.xcodeproj/project.pbxproj:36]: This is zombied and keeps coming back.  It can be  removed.

**Comment by petermisik** [comment]
The .DS_Store keeps coming back!!!
I don't know why the .pbxproj file keeps conflicting.

**Comment by Pete-Hoch** [comment]
OK, how do I check this out to build it and try out the code?

Usually, I can just copy the branch name and do a checkout. For example:

git checkout -b pr-54 origin/<from string>

But that does not work at all with this PR. In fact, the PR branch does not even seem to be on the same repository as the source code? This is new to me, please advise.

**Comment by petermisik** [comment]
Going to generate a new pull request on another branch.

**Pull request closed without merging**